jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+   <j-tray-plugin
+     description='Queries an area encompassed by the viewer using a specified catalog and marks all the objects found within the area.'
+    :link="'https://jdaviz.readthedocs.io/en/latest/imviz/plugins.html'"
```

Comment:
This link will be properly updated once the document has a new subsection for this plugin.